src/main/java/com/bookstore/dto/user/UserRegistrationRequestDto.java

```diff
@@ -0,0 +1,36 @@
+package com.bookstore.dto.user;
+
+import com.bookstore.validation.FieldMatch;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.Data;
+
+@Data
+@FieldMatch(password = "password", repeatPassword = "repeatPassword")
```

Comment:
What if you will need to use this annotation for other fields not password and repeatPassword? should properties name be linked to password and repeatPassword?